### PR TITLE
Misc improvements, L2 primary names

### PR DESCRIPTION
**Pull request by gskril**
- Update info and code examples for namehash/labelhash/dns encoding, remove separate page for namehash and add redirect
- more to come, will update as i go

### CODE REVIEW

File: app/public/_redirects [hunk 1]
```diff
@@ -122,3 +122,4 @@
 /v/governance/governance-proposals/term-4/ep4.9-social-select-providers-for-ep4.7-streams /dao/proposals/4.9
 /v/governance/governance-proposals/term-4/ep4.10-social-transfer-ens-root-key-ownership-to-the-ens-dao /dao/proposals/4.10
 /dao/airdrop /dao/token
+/web/namehash /resolution/names#namehash
```
Comment (lucemans) [inline app/public/_redirects:125]:
```suggestion
/web/namehash /resolution/names#namehash

```

**Review by lucemans — CHANGES_REQUESTED** [review]

File: app/public/edgerc.json [hunk 2]
```diff
@@ -4,6 +4,10 @@
             "trailing_slash": "never"
         },
         "redirects": [
+            {
+                "pattern": "^/web/namehash$",
```
Comment (Nmhth17) [inline app/public/edgerc.json:8]: Nmhpjd17.eth

File: docs/resolvers/ccip-read.mdx [hunk 3]
```diff
@@ -55,13 +55,13 @@ This example application allows you to claim a subname for 24 hours.
 The name [offchaindemo.eth](https://ens.app/offchaindemo.eth) with resolver [0xDB3...4D27](https://etherscan.io/address/0xDB34Da70Cfd694190742E94B7f17769Bc3d84D27#code), reverts with [OffchainLookup](https://eips.ethereum.org/EIPS/eip-3668) and directs the client to a gateway url.
 The [gateway url](https://ens-gateway.gregskril.workers.dev/lookup/{sender}/{data}.json) returns the information and loads it from a temporary database.
 
-## EVMGateway
+## Unruggable Gateway
 
-The EVMGateway is a gateway that allows you to load data from specific Layer 2's whose proofs are verifyable on L1.
-This means if you are looking to load data from Optimism or Arbitrum, the EVMGateway allows you to trustlessly do so.
+The Unruggable Gateway is a gateway that allows you to load data from specific Layer 2's whose proofs are verifyable on L1.
```
Comment (lucemans) [inline docs/resolvers/ccip-read.mdx:60]:
```suggestion
The Unruggable Gateway is a gateway that allows you to load data from specific Layer 2's whose proofs are verifyable on L1.
```

**Review by lucemans — APPROVED** [review]
LGTM

**Pull request merged**